### PR TITLE
Add: fmt: year to publish date

### PR DESCRIPTION
**Pull request by uh-zz**
## Overview
Added year to date of generated images.

## Changes
Date.Format `Jan 2` to `Jan 2 2006`

### CODE REVIEW

**Comment by uh-zz** [comment]
@Ladicle ping

File: cmd/cmd.go [hunk 1]
```diff
@@ -196,7 +196,7 @@ func generateTCard(contentPath, outPath string, tpl image.Image, ffa *fontfamily
 		return err
 	}
 	if err := c.DrawTextAtPoint(
-		fmt.Sprintf("%s%s%s", fm.Author, cnf.Info.Separator, fm.Date.Format("Jan 2")),
+		fmt.Sprintf("%s%s%s", fm.Author, cnf.Info.Separator, fm.Date.Format("Jan 2 2006")),
```
Comment (Ladicle) [inline cmd/cmd.go:199]: Could you make this format configurable without changing the default behavior?

Comment (uh-zz) [reply cmd/cmd.go]: @Ladicle 
Time format can be set in config.

Comment (Ladicle) [reply cmd/cmd.go]: cool!

**Review by Ladicle — APPROVED** [review]

**Pull request merged**